xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -650,14 +650,41 @@ AVDictionary *CDVDDemuxFFmpeg::GetFFMpegOptionsFromInput()
 
       if (name == "seekable")
         av_dict_set(&options, "seekable", value.c_str(), 0);
+      // map some standard http headers to the ffmpeg related options
       else if (name == "user-agent")
       {
         av_dict_set(&options, "user-agent", value.c_str(), 0);
+        CLog::Log(LOGDEBUG, "CDVDDemuxFFmpeg::GetFFMpegOptionsFromInput() adding ffmpeg option 'user-agent: %s'", value.c_str());
         hasUserAgent = true;
       }
+      else if (name == "content-type")
```

Comment:
That why I stuck to the basic list. But, I will remove the content-type. I don't thing that ffmpeg will ever be sending POSTs.